invenio_app_ils/records/api.py

```diff
@@ -62,12 +62,21 @@ class Item(IlsRecord):
     def create(cls, data, id_=None, **kwargs):
         """Create Item record."""
         data["circulation_status"] = {
-            "$ref": "{scheme}://{host}/api/circulation/items/{pid_value}/loan"
-                    .format(
-                        scheme=current_app.config['JSONSCHEMAS_URL_SCHEME'],
-                        host=current_app.config['JSONSCHEMAS_HOST'],
-                        pid_value=data[cls.pid_field]
-                    )
+            "$ref": "{scheme}://{host}/api/circulation/items/{pid_value}/loan".format(
+                scheme=current_app.config["JSONSCHEMAS_URL_SCHEME"],
+                host=current_app.config["JSONSCHEMAS_HOST"],
+                pid_value=data[cls.pid_field],
+            )
+        }
+        if InternalLocation.pid_field not in data:
+            raise IlsException("Internal Location pid is required")
+
+        data["internal_location"] = {
+            "$ref": "{scheme}://{host}/api/internal-locations/{pid_value}".format(
```

Comment:
This works just because the host is different, otherwise it is exactly the same view registered by `invenio-records-rest`.
What about if we change this to something without the different host and:
`/api/refs/internal-locations/{pid-value}`
or
`/api/resolve/internal-locations/{pid-value}`